util/buildinfototags.py

```diff
@@ -0,0 +1,44 @@
+import pprint, requests, re, os
+import sys
+from xml.dom import minidom
+from optparse import OptionParser
+import json
+
+pp = pprint.PrettyPrinter(indent=4)
+
+# moar output, set = 1
+debug=0
+
+usage = "Usage: %prog [-n NAME] \n\n\
```

Comment:
not sure why you want curl. why not cat or anything else ? 

I'll update it to explicitly state it needs its input on standard input.